app/controllers/bulk_submissions_controller.rb

```diff
@@ -12,10 +12,13 @@ def index
 
```

Comment:
Submission Templates have been added to the bulk submissions page as a list. I'll tweak this to be a before filter.